packages/mantine/src/style.css

```diff
@@ -510,14 +511,23 @@
   padding: 0;
 }
 
+.bn-mantine .bn-table-cell-handle {
+  padding: 0 4px;
+}
+
 .bn-mantine .bn-table-handle svg {
   margin-inline: -4px;
 }
 
+.bn-mantine .bn-table-handle-not-draggable {
+  cursor: default;
```

Comment:
Instead of using `default` and `pointer`, maybe we should use `pointer` and `grab` instead? Slightly nicer imo